app/src/main/java/org/oppia/android/app/administratorcontrols/LoadProfileEditListener.kt

```diff
@@ -2,6 +2,6 @@ package org.oppia.android.app.administratorcontrols
 
 /** Listener for when an activity should load [ProfileEditFragment]. */
 interface LoadProfileEditListener {
-  /** This method is called when the fragment of [ProfileEditFragment] is inflated. */
+  /** Used for inflating [ProfileEditFragment] in a multipane fragment in tablet mode. */
```

Comment:
```suggestion
  /** Inflates [ProfileEditFragment] as part of a tablet mode a multipane fragment. */
```

We should describe what the method does, not when it's called.